conf/collector/fluentd/extract_log_level/extract_log_1_regex.conf

```diff
@@ -0,0 +1,88 @@
+<source>
+  @type tail
+  path "/var/log/containers/low-log-stress*.log,/var/log/containers/heavy-log-stress*.log"
+  exclude_path ["/var/log/containers/fluentd-*_openshift-logging_*.log", "/var/log/containers/elasticsearch-*_openshift-logging_*.log", "/var/log/containers/kibana-*_openshift-logging_*.log"]
+  pos_file "/var/log/containers.log.pos"
+  path_key path
+  refresh_interval 5
+  rotate_wait 5
+  tag  kubernetes.*
+  @label @MEASURE
+  #format /^.*(?<level>E[0-9]+).*$/
+  <parse>
+    @type multi_format
+    <pattern>
+      format json
+      time_format '%Y-%m-%dT%H:%M:%S.%N%Z'
+      keep_time_key true
+    </pattern>
+    <pattern>
+      format regexp
+       expression /^(?<time>[^\s]+) (?<stream>stdout|stderr)( (?<logtag>.))? (?<message>.*)$/
+      time_format '%Y-%m-%dT%H:%M:%S.%N%:z'
+      keep_time_key true
+    </pattern>
+  </parse>
+</source>
+
+<label @MEASURE>
+
+  <match kubernetes.**>
+    @type rewrite_tag_filter
+
+    <rule>
+      key message
+      pattern /^.*(?<level>\"message\":.*(Error|ERROR|E[0-9]+|level=error|Value:error|\"level\":\"error\")).*$/
```

Comment:
It make sense to also add another file that will have one regex, but much shorter one ... so it might be interesting to see if shorter and simpler regex takes is cheaper than more complicated regex

Reply:
@ajaygupta978 still need to address this one ... it is small but still might make sense to have such an option when we check things ...

Reply:
@eranra for this do you want me to run performance test also for smaller regex ?